gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/VertexTest.java

```diff
@@ -280,10 +280,9 @@ public void shouldHaveExceptionConsistencyWhenIdNotSupportedForAddEdge() throws
         }
 
         @Test
-        @FeatureRequirementSet(FeatureRequirementSet.Package.VERTICES_ONLY)
-        @FeatureRequirement(featureClass = VertexPropertyFeatures.class, feature = FEATURE_INTEGER_VALUES)
+        @FeatureRequirement(featureClass = Graph.Features.VertexFeatures.class, feature = Graph.Features.VertexFeatures.FEATURE_ADD_VERTICES)
         public void shouldHaveStandardStringRepresentation() {
-            final Vertex v = graph.addVertex("name", "marko", "age", 34);
+            final Vertex v = graph.addVertex();
```

Comment:
Yeah, I thought about just adding the `FEATURE_INTEGER_VALUES` annotation, but it seemed to me like the test didn't have anything to do with properties. I can add another test to test with `toString()` with properties.